### PR TITLE
add a note about required class for Alert

### PR DESCRIPTION
**Pull request by garak**
See #407

### CODE REVIEW

File: v5.html [hunk 1]
```diff
@@ -209,7 +209,8 @@ <h3>Alert Events</h3>
             <section id="alertDATA">
               <h3>Alert DATA API</h3>
               <p>The component will initialize all elements with proper DATA API found in the DOM. Note that the
-                <code>data-bs-dismiss="alert"</code> attribute is required for the triggering button.</p>
+                <code>data-bs-dismiss="alert"</code> attribute is required for the triggering button. Also note
+                that, differently from jQuery, the <code>show</code> class is required.</p>
```
Comment (thednp) [inline v5.html:213]: differently from the original version, the <code>show</code> class is required for its main method.

Comment (thednp) [reply v5.html]: typo "ist" -> "its"

Also: would you care to take an overall look at the components regarding these markup requirements?

Comment (garak) [reply v5.html]: I don't see any "ist"...

File: v5.html [hunk 2]
```diff
@@ -210,7 +210,7 @@ <h3>Alert Events</h3>
               <h3>Alert DATA API</h3>
               <p>The component will initialize all elements with proper DATA API found in the DOM. Note that the
                 <code>data-bs-dismiss="alert"</code> attribute is required for the triggering button. Also note
-                that, differently from jQuery, the <code>show</code> class is required.</p>
+                that, differently from the original version, the <code>show</code> class is required for ist main method.</p>
```
Comment (thednp) [inline v5.html:213]: "for ist main method"

**Pull request merged**